src/riak_core_claimant.erl

```diff
@@ -569,17 +569,15 @@ valid_resize_request(NewRingSize, [], Ring) ->
     %%            for dynamic ring, if all registered applications support it
     %%            the cluster is capable. core knowing about search/kv is :(
     ControlRunning = app_helper:get_env(riak_control, enabled, false),
-    SearchRunning = app_helper:get_env(riak_search, enabled, false),
     NodeCount = length(riak_core_ring:all_members(Ring)),
     Changes = length(riak_core_ring:pending_changes(Ring)) > 0,
-    case {ControlRunning, SearchRunning, Capable, IsResizing, NodeCount, Changes} of
-        {false, false, true, true, N, false} when N > 1 -> true;
-        {true, _, _, _, _, _} -> {error, control_running};
-        {_,  true, _, _, _, _} -> {error, search_running};
```

Comment:
Now that this `{error, search_running}` is no longer a possible return value here, we can also eliminate the corresponding clause in `riak_core_console:stage_resize_ring`.